test/views/meetings/meeting_list_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'Meeting List View', type: :view do
+  describe 'Student count display' do
+    it 'displays the correct count for a class with students and one teacher' do
+      teacher = create(:user, role: 'teacher')
+      students = create_list(:user, 5, role: 'student')
+      meeting = create(:meeting, members: students + [teacher])
+
+      render partial: 'meetings/meeting_list', locals: { meetings: [meeting] }
+
+      expect(rendered).to include("5")
```

Comment:
/pilot For this test, I want to validate that the number is in the proper column and not just that the number exists in the output.